src/zenml/utils/analytics_utils.py

```diff
@@ -87,6 +88,46 @@ def get_environment() -> str:
         return "native"
 
 
+def identify_user(user_metadata: Optional[Dict[str, Any]] = None) -> bool:
```

Comment:
I am a bit unsure how the analytics is working now. If the user gives the email do we associate it with the telemetry?

Reply:
Check it out: https://mixpanel.com/project/2277187/view/2823943/app/profile#distinct_id=052c534c-b426-4deb-a4f3-28e469418b59&